### DISCUSSION TITLE
How to use the discussion tab

### DISCUSSION THREAD

**Discussion by jbischof** [keras-team/keras-cv#1813] [General]
We've had a lot of great activity on the discussion tab lately! It's been great to hear from the community.

However, if your main goal is to request a feature or report unexpected behavior with the library, please file a "feature request" or "bug" on the Issues page. Issues are more closely monitored and easier to triage than a discussion thread if there is a clear deliverable.

Discussions can still be great for more generic topics such as
* understanding general capabilities of the library
* showcasing community projects
* questions about our roadmap
* questions about project administration

Thanks again! Look forward to hearing from you soon.

**Comment by hugopi** [comment]
Thank you for this clarification !
